Create postgresql-eol-check.md (#68)
percona-platform/portal-doc: docs/advisors/checks/postgresql-eol-check.md

# Percona Distribution for PostgreSQL End Of Life
## Description

> Your version of Percona Distribution for PostgreSQ reached End-of-life.

Reviewer: ```suggestion
This check notifies that your version of Percona Distribution for PostgreSQ has reached End-of-life.
```

# Percona Distribution for PostgreSQL End Of Life
## Description

> Percona Distribution for PostgreSQL, you are using reached End-of-life.

Reviewer 2: ```suggestion
Your version of Percona Distribution for PostgreSQ reached End-of-life.
```

Percona does not provide support or issue hotfixes for end-of-life releases.

> After that date, the software will be out of support, and users that will still be running that version will not

Reviewer 2: ```suggestion
```

> get any further updates or bug fixes.

Reviewer 2: I think the sentence above covers what we mean to say here. I would remove this sentence and not the one above because we don't actually mention any date.

## Resolution

> Please upgrade to the latest version of Percona Distribution for PostgreSQL according to our [update guide](https://docs.percona.com/postgresql/15/major-upgrade.html)

Reviewer 2: ```suggestion
Upgrade to the latest version of Percona Distribution for PostgreSQL, following the [upgrade instructions](https://docs.percona.com/postgresql/15/major-upgrade.html).
```
